### ISSUE TITLE
Switch "canonical" IDE for Orc projects to IntelliJ IDEA

### ISSUE THREAD

**Issue by jthywiss** [orc-lang/orc#221]
The Scala Eclipse IDE/plug-in [is officially dead](https://groups.google.com/forum/#!msg/scala-ide-dev/DCjbh5gRUOs/Vn9YFEW2BgAJ).  The Scala IDE of choice [appears to be IntelliJ IDEA](https://docs.scala-lang.org/getting-started-intellij-track/getting-started-with-scala-in-intellij.html), which is a reasonable choice.

Along with the switch to SBT #200, we should switch to IntelliJ IDEA as the "canonical" IDE for developing the Orc implementation.  (**Not to exclude other IDEs, though.**)

(This issue does _not_ address the OrcEclipse plug-in.)

**Comment by arthurp** [comment]
One major advantage of SBT is that there doesn't need to be a primary IDE since the IDE files are not committed and instead generated from the SBT build config. I think we should strongly prefer this approach since it avoids a lot of IDE config issues as well. And it's how most Scala projects do it.

**Comment by jthywiss** [comment]
"correct"  :)

**Comment by arthurp** [comment]
Yes but this issue becomes a no-op once we switch to SBT. So I'm not sure what it's for.

**Comment by jthywiss** [comment]
1. Set up generation of said IDE files from sbt. 
2. Update our [Quick Start to Developing the Orc Language Toolset](https://orc.csres.utexas.edu/quickstart.shtml) page.
